docs/source/release/v5.1.0/muon.rst

```diff
@@ -5,12 +5,13 @@ MuSR Changes
 .. contents:: Table of Contents
    :local:
 
-.. warning:: **Developers:** Sort changes under appropriate heading
-    putting new features at the top of the section, followed by
-    improvements, followed by bug fixes.
+Muon Analysis 2 and Frequency Domain Analysis
+#############################################
 
-Muon Analysis 2
-################
+.. figure:: ../../images/MuonAnalysis_release_5_1_0.png
```

Comment:
As for other pages, is there a bullet point that we can highlight in bold for this is or is it general across the new features for this interface???

Reply:
This one is just a general image.